src/modules/UI/Settings/reducer.js

```diff
@@ -28,6 +28,15 @@ export const settings = (state = initialState, action) => {
       loginStatus
     }
   }
+
+  case ACTION.SET_CUSTOM_TOKENS: {
+    const {customTokens} = data
+    return {
+      ...state,
+      customTokens
```

Comment:
why are you storing this data in redux?

Reply:
They need to be stored in Redux because are account-specific but we do not want to always re-query the disklet in order to obtain them.